addons/sale_crm/migrations/8.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,31 @@
+---Fields in module 'sale_crm'---
+sale_crm     / account.invoice          / section_id (many2one)         : module is now 'sale' ('sale_crm')
+sale_crm     / res.users                / default_section_id (many2one) : module is now 'sales_team' ('sale_crm')
+# Nothing to do - fields moved and not changed
```

Comment:
'sale_crm' doesn't depend on 'sales_team', so it is not assured that this field is going to be kept before sales_team is marking it.